specs/core/0_beacon-chain.md

```diff
@@ -1343,7 +1366,8 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
             withdrawable_epoch=FAR_FUTURE_EPOCH,
             initiated_exit=False,
             slashed=False,
-            high_balance=0
+            high_balance=0,
+            exposed_randao_reveals=[],
```

Comment:
I dislike the idea of the validator record object not being constant-sized; it intuitively feels like a good invariant to keep that it's constant. In other places (eg. phase 1), we're keeping challenge records as separate lists too; would it make sense to make it be a separate list here too?

Reply:
I changed this now to be a separate list. To make it a bit more efficient I use the epoch as a primary index, so in the typical case it should have very little overhead.